src/msg/async/net_handler.cc

```diff
@@ -34,6 +35,7 @@ namespace ceph{
 int NetHandler::create_socket(int domain, bool reuse_addr)
 {
   int s, on = 1;
+  IGNORE_UNUSED(on);
```

Comment:
> If no option value is to be supplied or returned, optval may be NULL.

see https://linux.die.net/man/2/setsockopt. so let's pass `nullptr` to `setsockopt()` instead.